std/algorithm/searching.d

```diff
@@ -2557,6 +2557,17 @@ template canFind(alias pred="a == b")
     assert( canFind!((string a, string b) => a.startsWith(b))(words, "bees"));
 }
 
+/// Search for mutliple items in an array of items (search for needles in an array of hay stacks)
```

Comment:
Still a typo here: multiple

Also it's probably a good idea to use multiple lines here as the soft limit is 80.

Reply:
Ah, I can't even spell multiple properly! I need to enable spell check "only" for comments.

Reply:
Oh and sorry for just noticing now, haystack is one word: http://dictionary.cambridge.org/english/haystack